lib/http_io.dart

```diff
@@ -32,7 +32,7 @@ import 'dart:typed_data';
 import 'src/char_code.dart';
 import 'src/crypto.dart';
 import 'src/http_exception.dart';
-import 'src/http_headers_impl.dart';
+import 'src/http_headers_impl.dart' hide parseCookieDate;
```

Comment:
Does `parseCookieDate` conflict with some other symbol in this file? Otherwise, I'm not sure it needs to be hidden here. It already won't be exported to importers of http_io.dart, right?

Reply:
Since `parseCookieDate` is a top level method in `http_headers_impl.dart` it would be visible to anyone who imports `http_io.dart`.

Reply:
I think not unless it is explicitly exported?

Reply:
Ah, you're right. I should look into when you do need `hide` since I have an inkling that only classes and enums are exported by default...